server/recceiver/test_cfstore.py

```diff
@@ -29,7 +29,7 @@ def test_3_iocs(self):
             6: ("ch6", "stringin"),
             7: ("ch7", "longout"),
         }
-        deferred = yield self.cfstore.commit(TR1)
+        _ = yield self.cfstore.commit(TR1)
```

Comment:
I'm a bit surprised by this. Wouldn't it be better to just do
```py
yield self.cfstore.commit(TR1)
```
?

Reply:
Yes. Probably. Tempted to just remove this test file entirely as it is sooo out of date and incompatible with other cfstore changes.